templates/init/dataconnect/queries.gql

```diff
@@ -1,56 +1,82 @@
-# # Example queries for a simple email app.
+# # Example queries for a simple movie app.
 
 # # @auth() directives control who can call each operation.
-# # Only admins should be able to list all emails, so we use NO_ACCESS
-# query ListEmails @auth(level: NO_ACCESS) {
-#   emails {
-#     id, subject, text, sent
-#     from {
-#       name
-#     }
+# # Anyone should be able to list all movies, so the auth level is set to PUBLIC
+# query ListMovies @auth(level: PUBLIC) {
+#   movies {
+#     id
+#     title
+#     imageUrl
+#     genre
 #   }
 # }
 
-# # Only admins should be able to list all users, so we use NO_ACCESS
+# # List all users, only admins should be able to list all users, so we use NO_ACCESS
 # query ListUsers @auth(level: NO_ACCESS) {
-#   users { uid, name, address }
+#   users { id, username }
 # }
-  
-# # Logged in users should be able to see their inbox though, so we use USER
-# query ListInbox @auth(level: USER) {
-#   # where allows you to filter lists
-#   # Here, we use it to filter to only emails that are sent to the logged in user.
-#   emails(where: {
-#     users_via_Recipient: {
-#       exist: { uid: { eq_expr: "auth.uid" }
-#     }}
-#   }) {
-#     id subject sent
-#     content: text # Select the `text` field but alias it as `content` in the response.
-#     sender: from { name address uid }
-#     # <field>_on_<foreign_key_field> makes it easy to grab info from another table
-#     # Here, we use it to grab all the recipients of the email.
-#     to: recipients_on_email {
-#       user { name address uid }
+
+# # Logged in user can list all their reviews and movie titles associated with the review 
+# # Since the query requires the uid of the current authenticated user, the auth level is set to USER
+# query ListUserReviews @auth(level: USER) {
+#   user(key: {id_expr: "auth.uid"}) {
+#     id
+#     username
+#     reviews: reviews_on_user {
```

Comment:
Consider adding a comment explaining aliasing and a comment explaining `<field>_on_<foreign_key_field>`, like the old example did.